### PR TITLE
Add Dapr 0.6.0 Release note

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Add Dapr 0.6.0 Release note

## Issue reference

#1346 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation
* [ ] Specification has been updated
* [ ] Provided sample for the feature

### CODE REVIEW

File: docs/release_notes/v0.6.0.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
+
+# Dapr 0.6.0
+ 
+We're happy to announce the release of Dapr 0.6.0! 
+
+**Highlights**
+
+* Introduced *new* liveness/readiness probe endpoint for Dapr system/sidecar as well as actor service health API in User application
```
Comment (shalabhms) [inline docs/release_notes/v0.6.0.md:8]: The endpoint is for Dapr Side car container health only.

File: docs/release_notes/v0.6.0.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+* Improve e2e test coverage and release process for manifest yaml
+* Fixed major/minor bugs
+
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
+
+If you're new to Dapr, visit the [getting started](https://github.com/dapr/docs/tree/master/getting-started) page and familiarize yourself with Dapr.
+
+Docs have been updated with all the new features and changes of this release. To get started with new capabilities introduced in this release, go to the [How To section](https://github.com/dapr/docs/tree/master/howto#how-tos).
+
+> **Note: This release contains a few minor breakings. See [breaking changes](#breaking-changes).**
```
Comment (shalabhms) [inline docs/release_notes/v0.6.0.md:22]: nit : ..a few minor breaking changes.

File: docs/release_notes/v0.6.0.md (hunk 1 above)
Comment (youngbupark) [reply docs/release_notes/v0.6.0.md]: fixed.

File: docs/release_notes/v0.6.0.md (hunk 2 above)
Comment (youngbupark) [reply docs/release_notes/v0.6.0.md]: fixed.

File: docs/release_notes/v0.6.0.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+### System service components
+
+**Added** required TLS connection for Kubernetes Operator
+
+**Added** required TLS connection for Placement Service
+
+**Added** required TLS connection for Sentry
+
+**Updated** client validation of Sentry to require TLS certificiation all the time
```
Comment (pruthvidhodda) [inline docs/release_notes/v0.6.0.md:112]: nit: certification

**Pull request merged**